File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // store an integer betwen 1 and 8, inclusive
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int line = 1; line <= height; line++)
    {
        // first pyramid
        for (int spaces = height - line; spaces > 0; spaces--)
        {
            printf(" ");
        }
        for (int hash = line; hash > 0; hash--)
        {
            printf("#");
        }

        // spacing
        printf("  ");

        // second pyramid
        for (int hash = line; hash > 0; hash--)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int


def main():
    height = get_height()

    for line in range(1, height + 1):
        # first piramid
        print(' ' * (height - line), end='')
        print('#' * line, end='')

        # spacing
        print('  ', end='')

        # second piramid
        print('#' * line)


def get_height():
    # gets an integer between 1 and 8, inclusive
    while True:
        h = get_int("Height: ")
        if h in range(1, 9):
            break
    return h


main()
